hr_holidays_credit/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Leave Credit",
+    "version": "13.0.1.0.0",
+    "category": "Human Resources",
+    "website": "https://github.com/OCA/hr",
```

Comment:
```suggestion
    "website": "https://github.com/OCA/hr-holidays",
```
Please change the website.

Reply:
Fixed.